tools/src/main/python/opengrok_tools/utils/command.py

```diff
@@ -403,11 +427,18 @@ def getretcode(self):
 
     def getoutputstr(self):
         if self.state == Command.FINISHED:
-            return "".join(self.out).strip()
+            s = os.linesep.join(self.out)
+            if len(self.out) > 0:
```

Comment:
Just `if self.out:` should work here as well, but it's not a big deal.

Reply:
I see, thanks !